src/services/DBService.js

```diff
@@ -36,37 +35,49 @@ module.exports.retrieveAccessToken = (teamId) => {
 	});
 };
 
-module.exports.storeJiraIssueID = (issueID, channel) => {
+module.exports.updateChannelContext = (channel, value, valueType) => {
```

Comment:
I think this would be neater with an object as the argument, e.g.
`updateChannelContext(channel, { projectKey: "POET" });`
Might be worth considering if it's possible to handle more than one at once.

Reply:
Simplest solution is probably to have an array of expression statements (eg "issueID = :id") that you accumulate using [].push(), and accumulate the attributeValues using Object.assign().

Reply:
Great suggestion, I've modified it to be along those lines of modularity and cleanliness 🔥